.github/workflows/deploy-prod-end.yml

```diff
@@ -0,0 +1,47 @@
+name: Hide prod maintenance page after deployment.
```

Comment:
The idea behind splitting this up is that there are manual steps involved in prod deployments, so it doesn't make sense to burn through all of the steps and then take down the maintenance page and post a completion notification as that would likely finish before the manual steps are completed.

Instead, the deployer would run the prod deploy start script, wait for it to finish, then run any manual steps, then run the prod deploy end script.